server/controllers/ReportCategoryController.js

```diff
@@ -0,0 +1,200 @@
+import db from '../database/models';
+
+const { ReportCategory } = db;
+
+/**
+ * @class ReportCategoryController
+ *
+ * @export {class}
+ */
+class ReportCategoryController {
+  /**
+   * @description - Create New ReportCategory
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @param {object} next call next funtion/handler
+   *
+   * @memberOf ReportCategoryController
+   *
+   * @returns {object} response JSON Object
+   */
+  static createReportCategory(req, res, next) {
+    const { title, description } = req.body;
+
+    return ReportCategory
+      .create({
+        title,
+        description,
+      })
+      .then((newReportCategory) => {
+        res.status(201).send({
+          reportCategory: {
+            message: 'New category created successfully',
+            id: newReportCategory.id,
+            title: newReportCategory.title,
+            description: newReportCategory.description
+          }
+        });
+      })
+      .catch(next);
+  }
+
+  /**
+   * @description - Get a Specific ReportCategory
+   * @static
+   *
+   * @param {object} req - HTTP Request
+   * @param {object} res - HTTP Response
+   * @param {object} next call next funtion/handler
+   *
+   * @memberOf ReportCategoryController
+   *
+   * @returns {object} response JSON Object
+   */
+  static getReportCategory(req, res, next) {
+    ReportCategory.findOne({
+      where: {
+        id: req.params.id
+      },
+      attributes: ['title', 'description']
+    })
+      .then((reportCategory) => {
+        if (!reportCategory) {
+          return res.status(404).json({
+            errors: {
```

Comment:
I think the error message does not need to be in an errors object going by the way we return our other error messages.

Reply:
Since we are using validatorjs and the errors are returned as shown in the image below, I do not see anything wrong here. 
Can you please clarify?
<img width="464" alt="screen shot 2018-10-16 at 8 28 34 pm" src="https://user-images.githubusercontent.com/7892780/47042034-33cfa500-d182-11e8-9d6b-3b9629763a6c.png">

Reply:
```suggestion
            return res.status(404).json({
            message: 'Report category not found'
          });
```
We should follow same pattern for 404 errors